lib/terraforming/resource/iam_user.rb

```diff
@@ -16,23 +16,23 @@ def initialize(client)
       end
 
       def tf
-        apply_template(@client, "tf/iam_user")
+        apply_template(@client, 'tf/iam_user')
       end
 
       def tfstate
         iam_users.inject({}) do |resources, user|
           attributes = {
-            "arn"=> user.arn,
-            "id" => user.user_name,
-            "name" => user.user_name,
-            "path" => user.path,
-            "unique_id" => user.user_id,
+            'arn'=> user.arn,
+            'id' => user.user_name,
+            'name' => user.user_name,
+            'path' => user.path,
+            'unique_id' => user.user_id,
```

Comment:
What is this warning for?  This is intentional, so that the next person who adds another member to this hash doesn't have to append a comma to this line and thus earn the `git blame` for the line.